theforeman.org/yaml/jobs/pipeline/katello-rpm-pipeline.yaml

```diff
@@ -20,3 +20,4 @@
     version:
       - '4.9'
       - '4.10'
+      - '4.11'
```

Comment:
This should be covered:
https://github.com/theforeman/jenkins-jobs/blob/b993393b01c86082cefda9cfea5dfbba5abd0d3c/branch-foreman#L26